README.md

```diff
@@ -1,71 +1,68 @@
 
-### True control and worker plane separation
+### Enhanced High Availability
 
-Using k0smotron the clusters controlplane and workerplane are truly separated. The controlplane, running on an existing cluster has no direct networking connection to the workerplane. This is similar patter nhow all the major cloud providers separate the control and worker planes on the managed clusters. 
+k0smotron allows you to leverage the power of SQL databases as a data store for your control planes through Kine. This flexibility means you can choose from a variety of data storage operators like Postgres, MySQL, or your cloud provider's managed databases. This offers a robust, high-availability solution, eliminating the dependency on Etcd.
 
-### Bring your own workers
+### Bring Your Own Worker Nodes
 
-With k0smotron you can connect worker nodes from ANY infrastructure to your cluster control plane. 
+k0smotron prioritizes flexibility in the integration of worker nodes and allows easy connection or creation of nodes. This ensures node isolation and flexible scaling, minimizing interference with the control plane. k0smotron operates with or without the ClusterAPI, offering you the freedom to select your preferred operational mode.
 
 ## How does it work
 
-You install k0smotron operator into an existing Kubernetes cluster. k0smotron operator will create and manage k0s control planes in that cluster. It leverages the natural pattern of working with custom resources to manage the lifecycle of the k0s control planes. k0smotron will automatically create all the needed Kubernetes lower level constructs, such as pods, configmaps etc., to 
-
-k0smotron is an Kubernetes operator designed to manage the lifecycle of k0s control planes in a Kubernetes (any distro) cluster. By running the control plane on a k8s cluster we can enjoy and leverage the high availability and auto-healing functionalities of the underlying cluster, a.k.a Mothership.
+The k0smotron controller manager is a service that will be installed into an existing Kubernetes cluster. k0smotron will create and manage k0s control planes in the management cluster like workloads. It leverages the natural pattern of working with custom resources to manage the lifecycle of the k0s control planes. k0smotron will automatically create all the needed Kubernetes lower level resources, such as pods, configmaps, etc.
 
-TODO: Insert pic
+k0smotron is a service designed to manage the lifecycle of k0s control planes in a Kubernetes (any distro) cluster. By running the control plane on a k8s cluster we can enjoy and leverage the high availability and auto-healing functionalities of the management cluster.
```

Comment:
IMO this and the paragraph above are quite much overlapping.

Also explain a bit how this integrates with ClusterAPI.